packages/compass-components/src/components/file-input.tsx

```diff
@@ -244,15 +252,15 @@ function FileInput({
           title="Select a file"
           leftGlyph={<Icon glyph="AddFile" title={null} fill="currentColor" />}
         >
-          {buttonText}
+          <span className={buttonTextStyle}>{buttonText}</span>
         </Button>
       </div>
       {showFileOnNewLine &&
         values &&
         values.length > 0 &&
         values.map((value, index) => (
           <div className={removeFileLineStyles} key={value}>
-            {value}
+            <div>{value}</div>
```

Comment:
Do we always wrap values in div now? why not span? or is this needed this way for styles?

Reply:
That depends on the case, if you want to signal a block you go for `div` if is more inline `span`. I generally always use div for layouts.